src/rosteravatarframe.cpp

```diff
@@ -87,8 +87,9 @@ void RosterAvatarFrame::drawAvatar()
             av = avatar_icon;
         }
     }
+    av.setDevicePixelRatio(::devicePixelRatio(this));
     ui_.lb_avatar->setPixmap(av);
-    ui_.lb_avatar->setFixedSize(avSize,avSize);
+    ui_.lb_avatar->setFixedSize(avSize / ::devicePixelRatio(this), avSize / ::devicePixelRatio(this));
```

Comment:
This looks wrong. if previously it matched with avatar size. Now it's not.

Reply:
Roster avatar frame doesn't match the avatar size at all. Without this line, on screen where devicePixelRatio=2, I was getting only half of the frame filled up with the image.

Reply:
Ah that's probably because qt changes scaling operations too. Weird..
You just could to optimize it quite a bit by storing multiplied value into different var, but that doesn't matter here.